tensorflow_asr/featurizers/text_featurizers.py

```diff
@@ -446,6 +445,8 @@ def indices2upoints(self, indices: tf.Tensor) -> tf.Tensor:
         Returns:
             unicode code points transcript with dtype tf.int32 and shape [None]
         """
+        if self.upoints is None:
+            self.__init_upoints()
```

Comment:
@gandroz Does this affect tflite and performance? If so, we should pass the `enable_tflite=True` to `__init__` so that we create upoints before creating tflite with `indices2upoints` function

Reply:
I did not check tflite yet... I got 0 experience with it, but I could verify. For the performances, I could not say.... But by checking if the `upoints` are `None` ensure we initialize them only once and only if we need them. I noticed that when debugging, my GPU RAM was fulled even before any operation on the network starts, and that'a where I ended with this solution. I don't see why performances would be degraded